app/ide-desktop/lib/dashboard/src/authentication/src/authentication/components/login.tsx

```diff
@@ -36,4 +156,10 @@ function Login() {
   );
 }
 
+function parseUrlSearchParams(search: string) {
```

Comment:
should this be a separate function?
- only used in one place
- very short (so doesn't hurt to be inlined)